res/app/device-list/details/device-list-details-directive.js

```diff
@@ -277,6 +279,20 @@ module.exports = function DeviceListDetailsDirective(
         }
       })()
 
+      // check if new column needs a new scope or not
+      // and build accordingly
+      function buildColumn(col, device) {
```

Comment:
At some point we'll probably have to add paging... but I'd like to avoid that as long as possible.